deeppavlov/models/bert/bert_squad.py

```diff
@@ -1,366 +0,0 @@
-# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import json
-import math
-from logging import getLogger
-from typing import List, Tuple, Optional, Dict
-
-import numpy as np
-import tensorflow as tf
-from bert_dp.modeling import BertConfig, BertModel
-from bert_dp.optimization import AdamWeightDecayOptimizer
-from bert_dp.preprocessing import InputFeatures
-from bert_dp.tokenization import FullTokenizer
-
-from deeppavlov import build_model
-from deeppavlov.core.commands.utils import expand_path
-from deeppavlov.core.common.registry import register
-from deeppavlov.core.models.estimator import Component
-from deeppavlov.core.models.tf_model import LRScheduledTFModel
-from deeppavlov.models.squad.utils import softmax_mask
-
-logger = getLogger(__name__)
-
-
-@register('squad_bert_model')
```

Comment:
remove from requirements_registry.json and remove from the docs/apiref/models/bert.rst
Class is mentioned in the docs/features/models/bert.rst and docs/features/models/squad.rst